Cargo.toml

```diff
@@ -32,5 +32,5 @@ default-target = "x86_64-pc-windows-msvc"
 targets = ["x86_64-pc-windows-msvc", "i686-pc-windows-msvc"]
 
 [package.metadata.winrt.dependencies]
-"Microsoft.Windows.SDK.Contracts" = "10.0.19041.1"
-"KennyKerr.Windows.TestWinRT" = "1.0.21"
+"Microsoft.Windows.SDK.Contracts" = { blob = "./winmds/Contracts" }
+"KennyKerr.Windows.TestWinRT" = { blob = "./winmds/TestComponent" }
```

Comment:
We should only have to do this for the `Windows.Foundation.*.winmd` files in the SDK nuget package. Those are the only ones that the `winrt-rs` crate needs a dependency on. The tests can still rely on nuget.